### PR TITLE
Fix project card border

### PR DESCRIPTION
**Pull request by fel-w**
# Description

This PR fixes the issue of elements shifting while hovering over a project card, caused by an added border on hover, that was not initially present. It also adds CSS modules to the component.

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

## Trello Ticket ID

https://trello.com/c/hNswSbLu

## How Can This Be Tested?

Pull this branch and run it locally, go to the projects list and hover over different project cards. The elements should have stopped shifting.

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings

# Screenshots
## Before
Before border

## After
After border

### CODE REVIEW

File: src/components/ProjectCard/ProjectCard.module.css [hunk 1]
```diff
@@ -21,7 +21,8 @@
   height: 5rem;
   display: flex;
   flex-direction: column;
-  background-color: #f6fbe5;
+  /* background-color: #f6fbe5; */
+  background-color: #f1f1f1;
```
Comment (fel-w) [inline src/components/ProjectCard/ProjectCard.module.css:25]: @neelxie while changing to CSS modules I noticed that this component was using the CSS of another (MemberCard) and that's where it was picking the background color that I added here. I could not locate it, kindly advise.

Comment (neelxie) [reply src/components/ProjectCard/ProjectCard.module.css:25]: @fel-w  whatever you do for now, do not modularize the project card as it sprews a lot of other unknown bugs such as this. Work on your task without changing that file. Screenshot from 2021-08-16 09-31-15

Comment (fel-w) [reply src/components/ProjectCard/ProjectCard.module.css:25]: Noted, let me revert

**Review by neelxie — APPROVED** [review]
lgtm

**Review by khalifan-kfan — APPROVED** [review]
looks fine

**Pull request merged**